katalog/goldpinger/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+namespace: monitoring
+
+resources:
+  - rbac.yml
+  - deploy.yml
+  - sm.yml
+
+images:
```

Comment:
Usually we put the image tag directly in the manifests, any particular reason to specify it in the `kustomization.yaml`?

Reply:
I like this approach because it's easier yo change the image tag ;) I mean, kustomize.yml is a <20 lines file, deployment is usually bigger than that.

Also, you don't need to modify the deployment if it is a new image tag in a repository.